app-connector-client/server/app.js

```diff
@@ -43,15 +45,21 @@ function init(varkesConfigPath = null, currentPath = "", nodePortParam = null) {
         res.render('index', { appName: varkesConfig.name })
     })
     app.get("/info", function (req, res) {
+
         var info = {
             appName: varkesConfig.name,
+            connected: connection.established(),
+            insecure: connection.established() ? !connection.secure() : false,
+            connection: connection.established() ? connection.info() : {},
             links: {
                 logo: LOGO_URL,
                 localApis: LOCAL_APIS_URL,
                 remoteApis: REMOTE_APIS_URL,
                 connection: CONNECTION,
                 registration: BATCH_REGISTRATION,
-                events: EVENTS_URL
+                events: EVENTS_URL,
+                key: CONNECTION + "/key",
```

Comment:
Some here, that are details relevant only if there is a connection, so go to /connection

Reply:
removed them from info and used the connection endpoint